src/compute-client/src/command.rs

```diff
@@ -969,6 +993,9 @@ impl<T: timely::progress::Timestamp> ComputeCommandHistory<T> {
                 ComputeCommand::CancelPeeks { uuids } => {
                     live_cancels.extend(uuids);
                 }
+                update @ ComputeCommand::UpdateMaxResultSize(_) => {
```

Comment:
I'm not super confident that I did this correctly

Reply:
It's correct but you don't actually need to rebind command. You could have just use `command` directly instead of renaming it to `update`. But what you did matches the style of the other branches.